waltz-test/src/main/python/waltz_ducktape/tests/validation/smoke_test.py

```diff
@@ -44,6 +45,15 @@ def test_produce_consume_while_killing_a_server_node(self, num_active_partitions
         self.run_produce_consume_validate(lambda: self.simple_validation_func(validation_cmd, timeout),
                                           lambda: self._kill_a_server_node(num_active_partitions))
 
+    @cluster(cluster_spec=MIN_CLUSTER_SPEC)
+    @parametrize(num_active_partitions=1, txn_per_client=80, num_clients=2, interval=600, timeout=300, interruption_length=5000, no_interruptions=4)
```

Comment:
We should be consistent in terms of variable naming. We have `num_active_partitions` and `num_clients` but then we have `no_interruptions` as well, which I am assuming is passing the number of interruptions.